### PR TITLE
Adding Assignment Reports

### PR DESCRIPTION
**Pull request by soachishti**

### CODE REVIEW

File: AutoGrade/admin.py [hunk 1]
```diff
@@ -97,11 +111,11 @@ def get_queryset(self, request):
...
-    
+
 @admin.register(Submission) 
 class SubmissionModelAdmin(admin.ModelAdmin):
     def get_queryset(self, request):
         qs = super(SubmissionModelAdmin, self).get_queryset(request)
         if request.user.is_superuser:
             return qs
-        return qs.filter(assignment__course__instructor__user=request.user)
+        return qs.filter(assignment__course__instructor__user=request.user)
```
Comment (recluze) [inline AutoGrade/admin.py:121]: Your editor is probably removing the NL character from the the last line. I believe this is not good. (Not an issue for this PR but you should fix this. Just a nitpick =] )

**Review by recluze — APPROVED** [review]
Looks good for the base functions. I have some concerns about the code if it is to be used for general audience rather than just us but that can be filed as "enhancement" issues later. Let's merge.

File: AutoGrade/admin.py (hunk 1 above)
Comment (soachishti) [reply AutoGrade/admin.py:121]: Acknowledged.

**Comment by soachishti** [comment]
I was also concerned with assignment_report interface which is out of admin panel (its default interface). But this is the quickest solution I could come up with, now :) 

@recluze

**Comment by recluze** [comment]
It's fine since we have authentication. It's cleaner this way.

**Pull request merged**